ER Diagram 2.drawio

```diff
@@ -0,0 +1 @@
6jexeKcrItV/4SwNB9Bm+Vxo+wCznT4mjCCZZ+LJ9bnrhEqaLML7HeY6XrBF/y9+rD/W6/UPOwyhcxETmkXuhlOti9zbAJscyKFzDKM3Re0nEQPqE8BLl6YY0YWdvgc0shvWZ2yk7Xu8NEEyYLCgbn2Ezw2dGvyi+fG8X5AMzjRPMxJTMJIZL2UjIG+ZbjFP8hrj+YryzmpJKmShLoBfGiwf0Sl/A3ku+sneiIkw09xptO04Q+j6KKbg4hzl8KSwwwWGcb9/ZuSf/iGrmxp1z45BnmpNjsD8m/2jzNJ/jmDwmDLcwIWIma0RNpQKoKRi7LaNb26mOQ84Q5j7xGMC8Xev4WrIbCChGvwDAXWHqmJoxtSVMoe8Tt5iNqJ6PqjvtD9Wv4AmZ8FuweXyaf/rn2x/e7PvDrQ0k+JBPGAs7JCoK8ALHMPq4lxLFr2If+Uzt+zYPGCcsGP6L8nzDNAhXOaYROF/yqE20mG6e6PV3jjHjgm/07J0xs7jgNx59d0eb8tEXlIZEBTRqMqH/gfKzvV0RCY277PzuNem7HeQ7XBV4lXqoBmWH8UYS71GdNdhq6FMUwTz8UX0OFbDbS8lLwU2pATPy/Td/oYJSuLfMarh3BRontndmNc3Jh90D7C2seJPzXYkjuRI/JQpJZVcy0sTeaOKkajbAbBhwuAtrPeC4kpVEIdGAh57j1fKF6GGMO2KHuh6GOD0IbkK848gUL8BWO1Ocydhij4Q8opfRwWtz8Pa06uBtu5mZAMvoyE54Tm5MAzRyBLPGiA/FyQM5HTiOGS+GVbt/B3L6bkzvXAhqn4kANagyI8s3yYjpBZgW+XNt2R1Xgq+P5M57mD+xy+lnmtah0OyO9jkdelBJ6TzdlLJDpatqU0FF1ueFkMy3StoHtJv2ASyrfTTvAw6YRbeJn4nI7+rzPmJzYMwEc+sg8QPkiYEsQFE+pn50jgxcIJhCQ7fFiUj7sUhOEK7RSxaOeQFVXxrkeEAZjqbXE456DCuzNqKF7N+FWX8AhGH8LoyxqwRgT3P1Sqx5+rjUh7Wgf2RaKAtgQq+MwvjtKnC1ZkI1R1HecQTYU/nEdFZLEI62Lx6sW0YhJxtJIPZW6Bmni5FUDIdUTC3NpMKUuSeMwyWMRisZTlLa4s5MX1balAtdVityxS/APM/JSpuDpqHqR5aHF2Oq60JQtSel+RcrZh1HYC8AVnti2pKBhWkaksNnf6wUuAzcXjPUB+aAteSot8xMGBhuZaWhYcMkgIL/NKsMlMaZrY8VgVjT1c1IUbzNsYrD2ubVYeLxVzLNO6P0h/cB/uy7YXobWQ41kLYO6z0rqXFODsM2Gxo0V3t7Fn1Z6aAWWE50Kl4Esyz0mruV9uCaDQot/tgidUPyoItbdhrg5csqo3G0VAfOpKXS8Pt1EObokUR0etU6hUkVMmmMW6UNZDi8cyvtjHYdo+q8TEsx2jUV4XfS1WB3MvslO4rTNFC33lHOCrBiSb/p1EdYsT1PlrSViVWvJLH0hML9SpKbUulAce544UDP8wIq21Mq1LJ6cspqOB3NcLqGVQH0zphOj4J6xsKgfpE+XAzft1PhGRu1T+lo4Y9cELrG6VsS0fj8c8X6IkHN9WvqjvWOllhf7tEVD20c6cwnkoSaRX9n9V136FRa6bUVpbl6QO9vMWZToLpJsZgCpb+VJqbamo4X7mNNaylgffNuGKCihHiIxndurakPs2D7XC0XADV3NZcSwgOjBSFSufUZtfbqhmq1UV4cTOBXzuNcN0Pgic+ij04UUy29MgR3opkhmFalTx4bxLVBEZRjhvM7s9u4SHzaNnE4L7Vgq2nqobjiALeufTfDBh7MBp2lapmANjYkRxcBVcco7S7kigYZjTEe2vSKXEqXpzDOXpXrOK47SAtE2pzpHsbPtCTmKuS3tbh7RuL2/N7GkTva29zWe9uB2CuUyFodDR3FGG/1sccP13aZx2/LdSYRLXF5ScmnBf0El8Q28+cEKmosG7qCaFu8U+MIqkU+oK0Un1DxbjkK32AofINYsd+eb5AdM1c4UUHMVV4GgtynfErGh0v88IcoKl9YgbH2q4m48l3Xhbm4ylIz4sXdro8b7yPAaXmYCyLArCnfmgyKbxVlesOYW+2DVCuSbZ0iDoxugv6lsdqegrL5yPnn+vbdJHr5LtzDMMcTpvo7X8ypsjf18khHp0Phj1nZmAcnP1/FlTh8sw0gB+xeh2+Fo/mFiLE9UWi9X5rE14leh8saUAQFfBOvBtueDCsVCRTbj/FetlINheQxD7lv0TJFMKNzUGLb9c5+r6uHitPWE0MeuipX53bXQbUkjvXmf4seM+AEsJoAyskdPe61yxxjO7tAqCBWM0L3QkSbFag4brMkY2sbNCg28/kR0r18fnau6ajWsnfFNdUFOVpyQ53t5XbOTj7ndWWlNhunEgax/RtgxRy69n+rU2HJFTx+mX+Q/cC4/4bKe6n335AclMK6D/osaes31RRGV9u0qH3WVaU6+8wtWX1Vul6En3ll1PDEINXdapHm8HZTBmqL5YhNNxZriylacuUnXseK3/nonCa2QQYlbbq62aClPQUm/dTTZHLUv56xoqtFzmc27ZR6fuupCNeD+K2nWg2WevXfSE6XjZSvL8p3awr1/o1/6amO87HbfUVeDuMF0cF+MkBcjMwnn45NBoh7zkC6T3QMc3RPfVJ2YQhSL3rVNGDuz59dOu14wLEIu7J0VMh24i/bnbDPTGveTk52fdhOrCpYzOjx+vJ4tjjKHT1eQQpNyWD/kkyVbalWR5kpQKEHow/sxDL0/Z13pGZzcJs0uhMadYgZQ7wdwG2h59tAxltFvc3OqLc85/9xmUR4g8ZfRtfnF8DoFg4SIXnTxet3C4L+C36gyyvY8tDo89Ur2RESH4oKaatXJcuL0q/fkh0xwLlygVW/pizP7Fy/Kd+Kxf7A7cyYyWGKcV724ilMgs/YR7TF/w==</diagram></mxfile>
```

Comment:
Damn that looks perfect!

Reply:
Might want to upload the updated png as well so we can see it hahahaha @robertfmoore